tests/integration_tests/performance/test_block_performance.py

```diff
@@ -393,3 +395,81 @@ def test_block_vhost_user_performance(
         fio_block_size,
         vm,
     )
+
+
+@pytest.mark.nonci
+@pytest.mark.parametrize("vcpu_count", [1, 2], ids=["1vcpu", "2vcpu"])
+def test_vhost_user_block(microvm_factory, guest_kernel, rootfs, vcpu_count, metrics):
+    """
+    This test simply tries to boot a VM with
+    vhost-user-block as a root device.
+    """
+
+    vhost_user_socket = "/vub.socket"
+
+    # Magic number 5 picked referring to test_memory_overhead.py
+    # because it helps in ABfying the test
+    for _ in range(5):
+        # logic to test vhost-use picked from test_vhost_user_block()
+        vm = microvm_factory.build(guest_kernel, None, monitor_memory=False)
+
+        # Converting path from tmpfs ("./srv/..") to local
+        # path on the host ("../build/..")
+        rootfs_path = to_local_dir_path(str(rootfs))
```

Comment:
this is being reworked in https://github.com/firecracker-microvm/firecracker/pull/4247

Reply:
removed.